### ISSUE TITLE
Model->fill() don't work if field is translatable

### ISSUE THREAD

**Issue by ref** [OFFLINE-GmbH/oc-mall-plugin#614]
I make my custom products import script with artisan command.

Simplify method for sample:

```php
...
$importItems = json_decode(file_get_contents($importFile), true);

foreach ($importItems as $item) {
    $product = Product::firstOrNew([
        'user_defined_id' => $item['article_no'],
    ]);

    $product->fill([
        'name' => $item['title_en'],
        'slug' => $slug,
        'additional_properties' => $someArray,
    ]);

    $product->save();
}
...
```

If table `offline_mall_products` empty, first import work great.

But if `offline_mall_products` has entries, then the field 'additional_properties' is not updated for an existing record. 

$product->fill() work correctly if I delete 'additional_properties' from array $translatable in model OFFLINE\Mall\Models\Product.

**Comment by tobias-kuendig** [comment]
This sounds more like an upstream problem with RainLab.Translate or October CMS itself. We just use a `$jsonable` property that is also translatable for the Product model. So there is nothing we can change in this plugin that would fix this issue.

Did you check out the respective GitHub repos if there is a similar issue already reported?

**Comment by tobias-kuendig** [comment]
Fixed in rainlab/translate-plugin#618
